src/libraries/Microsoft.PowerFx.Connectors/ConnectorFunction.cs

```diff
@@ -590,21 +591,22 @@ internal async Task<FormulaValue> InvokeInternalAsync(FormulaValue[] arguments,
             EnsureInitialized();
             runtimeContext.ExecutionLogger?.LogDebug($"Entering in {this.LogFunction(nameof(InvokeInternalAsync))}, with {LogArguments(arguments)}");
             BaseRuntimeConnectorContext context = ConnectorReturnType.Binary ? runtimeContext.WithRawResults() : runtimeContext;
-            ScopedHttpFunctionInvoker invoker = new ScopedHttpFunctionInvoker(DPath.Root.Append(DName.MakeValid(Namespace, out _)), Name, Namespace, new HttpFunctionInvoker(this, context), context.ThrowOnError);
```

Comment:
>new HttpFunctionInvoker [](http://example.com/codeflow?start=149&length=23)

this is where HttpFunctionInvoker was created before. 
who creates it now?

Reply:
Will be in BaseRuntimeConnectorContext.GetInvoker & I'll fix it